src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -38,6 +44,12 @@ const ConversionTaskInputParameters = ({
   const [skipPackageCheckValue, setSkipPackageCheckValue] = useState(
     skipPackageCheck.default
   );
+  const [optionalRepositoriesValue, setOptionalRepositoriesValue] = useState(
+    optionalRepositories.default === 'None' ||
```

Comment:
How about making this more shorter.

```suggestion
    [ 'None', '' ].includes(optionalRepositories.default)
    ? [] 
    : optionalRepositories.default.split(',')
```

Reply:
Thanks for the tip. Done